docs/configuration/module-config/mqtt.mdx

```diff
@@ -8,7 +8,7 @@ sidebar_label: MQTT
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-If your device is connected to WiFi you can enable it to forward packets along to an MQTT server. This allows users on the local mesh to communicate with users on the internet.
+If your device is connected to WiFi you can enable it to forward packets along to an MQTT server. This allows users on the local mesh to communicate with users on the internet. At least one channel must be enabled as uplink and/or downlink for the MQTT module to work. See [channels](/docs/settings/config/channels#downlink-enabled)
```

Comment:
Strictly speaking this is not correct. The node will still send status messages, like 'i just rebooted and connected' just no protobuf content.

Reply:
also MQTT is enabled for wired ethernet too... :-)

Reply:
I see, perhaps we can paraphrase what I wrote to reflect that? This is something I hadn't realised either when I was testing.